### PR TITLE
[SYCL][Matrix] Making PVC tf32 test cases pass on PVC and removed the XMX8 TF32 test cases as they are not supported

### PR DESCRIPTION
**Pull request by arnamoy10**

### CODE REVIEW

File: sycl/test-e2e/Matrix/joint_matrix_tf32.cpp [hunk 1]
```diff
@@ -10,7 +10,7 @@
 // RUN: %{build} -o %t.out -DSYCL_EXT_ONEAPI_MATRIX_VERSION=4
 // RUN: %{run} %t.out
 
-// XFAIL:*
+// XFAIL:CPU
```
Comment (YuriPlyakhin) [inline sycl/test-e2e/Matrix/joint_matrix_tf32.cpp:13]: AFAIK, this check is case-sensitive
```suggestion
// XFAIL:cpu
```

**Review by YuriPlyakhin — CHANGES_REQUESTED** [review]

File: sycl/test-e2e/Matrix/joint_matrix_tf32.cpp (hunk 1 above)
Comment (arnamoy10) [reply sycl/test-e2e/Matrix/joint_matrix_tf32.cpp]: Thanks, done

**Review by YuriPlyakhin — APPROVED** [review]
LGTM

**Comment by aelovikov-intel** [comment]
> @intel/llvm-gatekeepers Help merge please? I think the error is due to removal of two existing files, which is giving rise to conflicts.

You have to resolve the conflicts first.

**Comment by arnamoy10** [comment]
> > @intel/llvm-gatekeepers Help merge please? I think the error is due to removal of two existing files, which is giving rise to conflicts.
> 
> You have to resolve the conflicts first.

@aelovikov-intel Conflicts resolved.  The failure seems non-related.  Help merge please?

**Comment by steffenlarsen** [comment]
HIP failures reported in https://github.com/intel/llvm/issues/10460.

**Pull request merged**